src/sources/socket/tcp.rs

```diff
@@ -72,7 +72,7 @@ pub struct TcpConfig {
 
     /// The size of the receive buffer used for each connection.
     ///
-    /// Generally this should not need to be configured.
+    /// Generally, this should not need to be configured.
```

Comment:
When would it need to be changed?

Reply:
For all of these flagged "Generally, this should not need to be configured."  , all for receive buffer sizes,

, I wonder if we should just remove those from the doc comment altogether.
At one point we had in the external docs, flags for common/uncommon configuration settings, but those don't exist anymore.